src/views/Graph.vue

```diff
@@ -224,9 +229,34 @@ export default {
     }
   },
 
-  setup () {
+  props: { initialOptions },
+
+  setup (props, { emit }) {
+    /**
+     * The transpose toggle state.
+     * If true layout is left-right is false it is top-bottom
+     * @type {import('vue').Ref<Bool>}
+     */
+    const transpose = useInitialOptions('transpose', { props, emit }, ref(false))
```

Comment:
The default value should be a normal type, not a ref
```suggestion
    const transpose = useInitialOptions('transpose', { props, emit }, false)
```